topics/rediscli.md

```diff
@@ -345,6 +345,22 @@ key, like in the following example:
     127.0.0.1:6379> ZADD<TAB>
     127.0.0.1:6379> ZCARD<TAB>
 
+Once you've entered a valid command name at the prompt, its syntax will be
+displayed by the CLI by default. The default syntax hints display can be
+controlled via the CLI's preferences.
+
+## Preferences
+
+The CLI's behavior can be customized to your preferences. Once started, the CLI
+loads the preferences from a file named `.redisclirc` in the home directory of
+the user. Preferences can also be set during runtime.
+
+To set a preference, either via file or in runtime, use the the special `:set`
```

Comment:
Perhaps: "To set preferences, use the special `:set` command.  The following preferences can be set, either by typing the command in the CLI or adding it to the `.redisclirc` file:"

Reply:
Accepted